onnxscript/backend/onnx_export.py

```diff
@@ -419,6 +428,51 @@ def _python_make_node(self, onnx_node, opsets, indent=0):
         ]
         return "".join(text)
 
+    def translate_function_signature(self, funproto: onnx.FunctionProto) -> str:
+        """Generate signature for FunctionProto."""
+        type_map = _attribute_param_types(funproto)
+
+        def attr_sig(attr_name: str) -> str:
+            name = self._rename_variable(attr_name)
+            # A default type of INT is used for attribute parameters that are never used.
+            type = type_map.get(attr_name, onnx.AttributeProto.INT)
+            typerep = onnxscript.type_annotation.onnx_attr_type_to_onnxscript_repr(type)
+            return f"{name}: {typerep}"
+
+        inputs = [self._rename_variable(x) for x in funproto.input]
+        attrs = [attr_sig(x) for x in funproto.attribute]
+        input_and_attrs = ", ".join(inputs + attrs)
```

Comment:
```suggestion
        input_and_attrs = ", ".join((*inputs, *attrs))
```

Reply:
I find this harder to understand: it is a more roundabout way to concatenate two lists.

Reply:
The warning message earlier should be addressed some other way. Will take a look at it.